extension/src/resources.ts

```diff
@@ -95,6 +95,20 @@ export function getExtensionSettingString(settingName: string): string {
     return setting.toString().trim();
 }
 
+export function getExtensionSettingBoolean(settingName: string): boolean {
```

Comment:
is this function used anywhere?

Reply:
Not right now, but it will be in the next PR. Here is the snippet of the next PR's information harvesting and its call to the decision maker.
```
const macFilter = AutoLispExt.Resources.getExtensionSettingBoolean('completion.FilterMacOS');
const acadVer = Number.parseInt(AutoLispExt.Resources.getExtensionSettingString('help.TargetYear'));
const isDCL = AutoLispExt.Documents.getSelectorType(document.fileName) === AutoLispExt.Selectors.dcl;
const winOkay = includeWindowsFunctions(os.platform(), isDCL, macFilter, acadVer);
``` typescript
Now that we have a release year targeting from my last PR, Mac users can now be included/excluded DCL functions in addition to VLA. So, I am layering in a lot more decision making under that checkbox and put 100% of that decision making into a single function taking a bunch of arguments so it can be easily tested.